packages/augur-test/libs/ContractAPI.ts

```diff
@@ -178,11 +185,18 @@ export class ContractAPI {
     await market.contribute(payoutNumerators, amount, "");
   }
 
-  public derivePayoutDistributionHash(market: GenericAugurInterfaces.Market<ethers.utils.BigNumber>, payoutNumerators: Array<ethers.utils.BigNumber>): Promise<string> {
+  public async getRemainingToFill(market: GenericAugurInterfaces.Market<ethers.utils.BigNumber>, payoutNumerators: Array<ethers.utils.BigNumber>): Promise<ethers.utils.BigNumber> {
+    const payoutDistributionHash = await this.derivePayoutDistributionHash(market, payoutNumerators);
+    const crowdsourcerAddress = await market.getCrowdsourcer_(payoutDistributionHash);
+    const crowdsourcer = this.augur.contracts.getReportingParticipant(crowdsourcerAddress);
+    return await crowdsourcer.getRemainingToFill_();
```

Comment:
I found this regarding return await: https://eslint.org/docs/rules/no-return-await

Reply:
Oh, interesting! We're actually doing this in a fair number of places throughout the code, but I'll update these.